### PR TITLE
added some comment

### PR DESCRIPTION
**Pull request by teukuTM**

### CODE REVIEW

File: api/routes/order.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// routes order
```
Comment (hadikurniawans) [inline api/routes/order.js:1]: good job

**Review by hadikurniawans — COMMENTED** [review]
good job tim

File: api/routes/order.js (hunk 1 above)
Comment (teukuTM) [inline api/routes/order.js:1]: hey how about i adding some code on your project?

**Review by teukuTM — COMMENTED** [review]
adding some code

**Pull request merged**